vispWrapper/visp-demo.h

```diff
@@ -1,19 +1,20 @@
-// On Microsoft Windows, use dllexport to tag symbols.
-# if defined(_WIN32) || defined(__CYGWIN__)
-#   define VISP_WRAPPER_EXPORT __declspec(dllexport)
-# else // On Linux
-#   define VISP_WRAPPER_EXPORT
-# endif // defined(_WIN32) || defined(__CYGWIN__)
+#define TESTDLL_API __declspec(dllexport)
 
 #include <visp/vpMatrix.h>
 #include <visp/vpMath.h>
 #include <visp/vpTranslationVector.h>
 
+#include <visp3/core/vpImage.h>
+#include <visp3/core/vpTime.h>
+#include <visp3/io/vpImageIo.h>
+#include <visp/vpImageConvert.h>
+

```

Comment:
Could you please make uniform the includes using everywhere `#include <visp3/....h>`